j2/vars/sample-rules.yaml

```diff
@@ -557,4 +555,125 @@ rules:
         fromVertexLabel: Object.Data_Source
         toVertexLabel: Event.Group_Ingestion
 
+  ADP:
+    vertices:
+      - label: Person.Natural
+        props:
+          Full_Name: "${Nome?.trim().toUpperCase()}"
+          Full_Name_fuzzy: "${Nome?.trim().toUpperCase()}"
+          Type:
+            val: "cliente"
+            excludeFromSearch: true
+#          Customer_ID: "${Customer_ID?.trim()}"
+          Date_Of_Birth:
+            val: "${Data_de_Nascimento_Idade}"
+            type: "java.util.Date"
+            mandatoryInSearch: true
+            predicate: "eq"
+#          Gender: "${Sex}"
+
+      - label:

```

Comment:
Extra label?

Reply:
Yeah sorry ... forgot to remove ... done!